.gitignore

```diff
@@ -99,3 +99,5 @@ db.sqlite3
 
 # vscode
 .vscode/
+# migrations
+migrations/
```

Comment:
Hello @Philipotieno ,
Since the team agreed on having initial migrations only in develop, adding this line of the code would mean no migrations would be pushed at all. You could remove it.

Reply:
ok changing them